pages/team-structure.js

```diff
@@ -0,0 +1,402 @@
+            },{
+                _id: 8,
+                name: "Team 4",
+                leader: {
+                    _id: 24,
+                    name: {
+                        first: "Jenny",
+                        last: "Ma"
+                    }
+                }
+            }
+        ]
+    },{ // Software
+        _id: 2,
+        projects: [
+            {
+                _id: 1,
+                name: "Team Hub",
+                leader: {
+                    _id: 8,
+                    name: {
+                        first: "Kevin",
+                        last: "Bai"
+                    }
+                }
+            },{
+                _id: 2,
+                name: "Embedded Software",
+                leader: {
+                    _id: 3,
+                    name: {
+                        first: "Ken",
+                        last: "Livington"
+                    }
+                }
+            },{
+                _id: 3,
+                name: "Website",
+                leader: {
+                    _id: 10,
+                    name: {
+                        first: "Jenny",
+                        last: "Ma"
+                    }
+                }
+            },{
+                _id: 4,
+                name: "POD Systems",
+                leader: {
+                    _id: 11,
+                    name: {
+                        first: "Jenny",
+                        last: "Ma"
+                    }
+                }
+            }
+        ]
+    }
+]
+
+const project_json = [
+    {
+        name: "Electrical Lead", 
+        leader: {
+            _id: 2,
+            name: {
+                first: "Steven",
+                last: "Xiong"
+            }
+        }
+    },{
+        name: "Mechanical Lead", 
+        leader: {
+            _id: 3,
+            name: {
+                first: "Ken",
+                last: "Livington"
+            }
+        }
+    }
+]
+
+// TODO: use this later
+const selectionTiers = [
+    {
+        id: 1,
+        label: "Directors"
+    },
+    {
+        id: 2,
+        label: "Team Leads"
+    },
+    {
+        id: 3,
+        label: "Project Head"
+    },
+    {
+        id: 4,
+        label: "Subordinates"
+    },
+];
+
+const teamHierCustomTheme = {
+    ...theme,
+    breakpoints: ["810px","1090px", "1200px"]
+};
+
+
+// TODO: this component should own the "Selected Member state field"
+const TeamHierParentContainer = styled(Card)`
+    border-radius: 0;
+    overflow-x: hidden;
+
+    flex-grow: 1;
+`;
+
+const HierSubSection = styled(SystemComponent)`
+    overflow: hidden;
+    margin-bottom: ${props => props.theme.space[5]}px;
+`;
+
+// This should be a molecule
+// state : not owned --> the particular level we are at 
+// previous and current selections ie) SW team, team hub
+const TeamStructSideNav = styled(SystemComponent)`
+    background-color: #FFFFFF;
+    margin: ${props => props.theme.space[4]}px;
+`;
+
```

Comment:
We should probably keep all components in separate files (unless its like a one-line functional component). Also, you can consider using React Hooks to keep this as a functional component.